### PR TITLE
Initial presets for baby care amenities

### PR DESCRIPTION
**Pull request by govvin**
Convenient presets for the following baby care amenities:
baby_feeding, diaper, power_supply

We're planning to use the OSM Contributor app, and this preset for a local initiative to promote mapping of baby feeding/nursing rooms.

### CODE REVIEW

File: presets/babycare.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  - name:
+      default: Baby Care group
+      en: Baby Care group
+    url: 'https://wiki.openstreetmap.org/wiki/Proposed_features/babycare'
+    items:
+      - name: baby_feeding=yes
+        url: 'https://wiki.openstreetmap.org/wiki/Proposed_features/babycare'
+        label:
+          default: Baby feeding
+          description:
```
Comment (nlehuby) [inline presets/babycare.yml:19]: you have an indentation issue here :wink:

File: presets/babycare.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                  default: no
+                  en: No
+              - yes:
+                  default: yes 
+                  en: Yes
+      - name: diaper=yes
+        url: 'https://wiki.openstreetmap.org/wiki/Proposed_features/babycare'
+        label:
+          default: Diaper changing table
+          description: This is a fixture to change a baby's diaper. Often placed in a public toilet, and usually wall-mounted, fold-down shelf, with a safety strap to prevent a wiggly baby from falling off.
```
Comment (nlehuby) [inline presets/babycare.yml:92]: you also have an indentation issue here, and the description value should be in the `default` line below.

File: presets/babycare.yml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+              default: Describes availability of power supply sockets.
+              en: Describes availability of power supply sockets.
+            values:
+              - yes:
+                  default: yes 
+                  en: Yes
+              - no:
+                  default: no
+                  en: No
+           required = false
```
Comment (nlehuby) [inline presets/babycare.yml:141]: this line is not valid ;)

File: presets/babycare.yml [hunk 4]
```diff
@@ -0,0 +1,147 @@
+name:
+  default: Baby Care amenities preset
+  en: Baby Care amenities  preset
+  version: '0.1'
```
Comment (nlehuby) [inline presets/babycare.yml:4]: The `version`, `author`, `schema` and `description` lines should not be indented.

**Review by nlehuby — COMMENTED** [review]
Thank you for helping us to make a more inclusive map :1st_place_medal: 

There are a few identation issues with your preset file.

Please let us know if you need to guidance to fix these issues :wink:

**Comment by govvin** [comment]
@nlehuby , I would be very happy for any assistance. 

I'm now trying to correct the issues you mentioned, and making other revisions. Do you know of any tools to make this more convenient?

**Comment by nlehuby** [comment]
The simplest way is to run `npm run start` in the project directory: it will check the validity of the preset for you and give you hint about things to fix.
You can also use a YAML syntax highlighting tool in your text editor to help you find indentation errors.

**Comment by govvin** [comment]
Working on a new revision.

**Pull request closed without merging**